spec/shared_stripe_examples/subscription_examples.rb

```diff
@@ -215,6 +215,18 @@ def gen_card_tk
       }
     end
 
+    it 'when attempting to create a new subscription with the params trial', focus: true, live: true do
```

Comment:
@alexmamonchik by `focus: true` you've disabled 90% of all tests in Travis builds (which will fail if you remove it), only 60 tests are being executed now. Where's one more place in specs which has `focus: true`.